docs/guides/LocalLLMs.md

```diff
@@ -1,49 +1,40 @@
 # Local LLM Guide with Ollama server
 
-## 0. Install and Start ollama:
-run the following command in a conda env with CUDA etc.
+Ensure that you have the Ollama server up and running.
+For detailed startup instructions, refer to the [here](https://github.com/ollama/ollama)
```

Comment:
```suggestion
For detailed startup instructions, refer to the [here](https://github.com/ollama/ollama)

This guide assumes you've started ollama with `ollama serve`. If you're running ollama differently (e.g. inside docker), the instructions might need to be modified.
```

Reply:
Regarding the `ollama serve` command, it was removed to prevent misunderstandings among users, as executing `ollama` on Linux automatically starts it in the background.